tests/testdata/delegatedscanning/image-build/sleepforever.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Runs forever and terminates gracefully.
+
+die_func() {
+    echo
+    echo
+    echo "SIG caught, exiting"
+    exit 1
+}
+
+trap die_func TERM INT
+
+echo "Sleeping forever... (SIGINT or Ctrl-C to exit)"
+
+sleep inf && wait
```

Comment:
is `wait` needed? `sleep` runs in the foreground, right?

Reply:
In my testing yes it was, the signals were not being handled nicely on all the images I tested this with unless the wait was there.